vendor/github.com/docker/libnetwork/drivers/windows/windows.go

```diff
@@ -440,6 +448,31 @@ func parseEndpointOptions(epOptions map[string]interface{}) (*endpointConfigurat
 	return ec, nil
 }
 
+func parseEndpointConnectivity(epOptions map[string]interface{}) (*endpointConnectivity, error) {
```

Comment:
this is purely the endpoint options that are passed to the driver which is transparent to libnetwork core. It is key-value pair for the drivers per endpoint. So, this is okay (these are essentially map[string]string ... but historically it was `interface{}` and changing APIs is a pain and hence we left it that way. Again, this is not related to this change per-se.